chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/certifiedProduct/CertifiedProductHandler2014.java

```diff
@@ -699,7 +699,7 @@ private void parseTestTools(PendingCertifiedProductEntity product, PendingCertif
 				ttEntity.setTestToolVersion(testToolVersion);
 				TestToolDTO testTool = testToolDao.getByName(testToolName);
 				if(testTool != null) {
-					if(testTool.isRetired() && product.getIcs().equals("false")) {
+					if(testTool.isRetired() && product.getIcs() == false) {

```

Comment:
In the parent class to this one I have a method "asBoolean". Would this have been of help to you? Users will often enter different boolean values 0, 1, true, false, tRuE, FALSE, etc and I tried to handle that variety